File: problemas/1000/1064.js
const numeros = require('fs')
  .readFileSync('/dev/stdin', 'utf-8')
  .split('\n')
  .map(x => parseFloat(x))
  .filter(x => x > 0);

console.log(`${numeros.length} valores positivos`);
const media = numeros.reduce((a, b) => a + b) / numeros.length;
console.log(media.toFixed(1));

File: problemas/1000/1064.dart
import 'dart:io';

void main() {
  final numeros = new List();

  for (int i = 0; i < 6; i++) {
    double x = double.parse(stdin.readLineSync());
    if (x > 0) {
      numeros.add(x);
    }
  }

  stdout.writeln('${numeros.length} valores positivos');
  final media = numeros.reduce((a, b) => a + b) / numeros.length;
  stdout.writeln(media.toStringAsFixed(1));
}